client/admin/brewUtils/brewLookup/brewLookup.jsx

```diff
@@ -46,12 +71,23 @@ const BrewLookup = createClass({
 				<dt>Share Link</dt>
 				<dd><a href={`/share/${brew.shareId}`} target='_blank' rel='noopener noreferrer'>/share/{brew.shareId}</a></dd>
 
+				<dt>Created Time</dt>
+				<dd>{brew.createdAt ? Moment(brew.createdAt).toLocaleString() : 'No creation date'}</dd>
+
 				<dt>Last Updated</dt>
 				<dd>{Moment(brew.updatedAt).fromNow()}</dd>
 
 				<dt>Num of Views</dt>
 				<dd>{brew.views}</dd>
+
+				<dt>Number of SCRIPT tags detected</dt>
+				<dd>{this.state.scriptCount}</dd>
 			</dl>
+			{this.state.checkForScript &&
```

Comment:
Could we possibly remove `checkForScript` state altogether and just use `scriptCount > 0` (or `this.state.scriptCount &&`, since 0 is falsy) here since this is the only place it is used?

Reply:
Removed checkForScript in latest commit.

Reply:
I had to change it to `this.state.scriptCount > 0` as using just `this.state.scriptCount` would result in a `0` in the output when the button was not displayed.